src/phpDebug.ts

```diff
@@ -12,6 +12,9 @@ import { Terminal } from './terminal'
 import { isSameUri, convertClientPathToDebugger, convertDebuggerPathToClient } from './paths'
 import minimatch = require('minimatch')
 
+let open = require('open')
+let appendQuery = require('append-query')
```

Comment:
This is built into Node, no need for a dependency. Just use `import { URL } from 'url'`